### PR TITLE
update to match wiki changes

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 1978

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6e624c3bde/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-55be034898/how-to/)

File: content/dedicated-hosting/linux-spheres-of-support-for-dedicated-and-managed-ops.md [hunk 1]
```diff
@@ -17,17 +17,19 @@ This article outlines our **Fanatical Support&reg;** of software and server conf
...
 - Ubuntu 14.04 LTS (Trusty Tahr)
 
 The following operating systems are supported only for Managed Operations:
 
+- Debian&reg; 9 (Stretch)
 - Debian 8 (Jessie)
 - Debian 7 (Wheezy)
+- Ubuntu 18.04 LTS (Bionice Beaver)
 
 **Note**: Rackspace makes every effort to align our support dates for operating systems with the manufacturer's support dates. If a manufacturer decides to shorten the support life of an operating system, Rackspace might be forced to end support sooner than originally anticipated. For full details on support life, see the [Rackspace EOL Terms](https://www.rackspace.com/information/legal/eolterms).
```
Comment (RackerKate) [inline content/dedicated-hosting/linux-spheres-of-support-for-dedicated-and-managed-ops.md]: Looks like there's an empty space at the beginning of this line.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cc617fb64c/how-to/).

**Pull request merged**